engine-api/src/main/java/pl/edu/icm/unity/engine/api/config/UnityServerConfiguration.java

```diff
@@ -350,6 +351,7 @@
 		defaults.put(CREDENTIAL_REQ_CONTENTS, new PropertyMD().setStructuredListEntry(CREDENTIAL_REQS).setList(false).setCategory(initCredReqCat).
 				setDescription("Credential requirement contents, i.e. credentials that belongs to it"));
 		
+		defaults.put(EMAIL_CONFIRMATION_REQUEST_LIMIT_OLD, new PropertyMD().setCategory(mainCat).setDeprecated());
```

Comment:
Some description with info on update would be useful (Unity will log warning when using deprecated config entries). Also please add support for deprecated entry: when defined it should be used.